### PR TITLE
[chore] redis ssl fix, worker

### PR DESCRIPTION
**Pull request by garguelles**
# Changes

- default worker
- redis ssl fix

### CODE REVIEW

File: saleor/settings.py [hunk 1]
```diff
@@ -623,6 +623,14 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
...
 CACHES = {"default": django_cache_url.config()}
 CACHES["default"]["TIMEOUT"] = parse(os.environ.get("CACHE_TIMEOUT", "7 days"))
+if "OPTIONS" in CACHES["default"]:
+    CACHES["default"]["OPTIONS"]["CONNECTION_POOL_KWARGS"] = {"ssl_cert_reqs": None}
+else:
+    CACHES["default"]["OPTIONS"] = {
+        "CONNECTION_POOL_KWARGS": {
+            "ssl_cert_reqs": None,
+        }
+    }
```
Comment (jsnbuchanan) [inline saleor/settings.py:633]: @garguelles is this correct? From what I glean it's setting the SSL cert requirement to `None` for both cases. What does this change?

**Review by jsnbuchanan — CHANGES_REQUESTED** [review]

File: saleor/settings.py (hunk 1 above)
Comment (garguelles) [reply saleor/settings.py:633]: yes, this just prevents the default.options from getting overwritten if it exist in the `default` object.

Comment (garguelles) [reply saleor/settings.py:633]: Setting ssl cert reqs to None because heroku handles the ssl cert for us

Comment (jsnbuchanan) [reply saleor/settings.py:633]: Okay, thank you for explaining.

**Review by jsnbuchanan — APPROVED** [review]

**Review by ajmiclat — APPROVED** [review]

**Pull request merged**